packages/terra-section-header/src/SectionHeader.scss

```diff
@@ -0,0 +1,92 @@
+@import '~terra-mixins';
+
+:local {
+  .section-header {
+    @include terra-padding-end(var(--terra-section-header-padding-end, 0.7142857143rem));
```

Comment:
@bjankord are we still using the mixing to define the paddings or does we need to define left this without these mixins?

Reply:
Isn't this part of the [_terra ecosystem_](https://github.com/cerner/terra-core/tree/master/packages/terra-mixins/docs#terra-mixins)?

Reply:
We typically rely on the [RTL postcss plugin](https://github.com/cerner/terra-core/blob/master/packages/terra-site/webpack.config.js#L92) for bi-directional styling. The bidi mixins in terra-mixins probably need to be deprecated. While we don't typically use them, we've kept them around in case any teams are currently using them.